### DISCUSSION TITLE
unsupported upgrade path 1.30->1.31

### DISCUSSION THREAD

**Discussion by erustyrannus** [siderolabs/talos#9260] [Q&A] [answered]
Hello, sorry for the noob question, I am trying to upgrade K8S to the latest version but I am getting the following response:

talosctl --nodes <controlpane> upgrade-k8s --to 1.31 --dry-run
automatically detected the lowest Kubernetes version 1.30.4
unsupported upgrade path 1.30->1.31 (from "1.30.4" to "1.31")

Is this related to My talos version, should I wait for a new talos Release. Thanks.

**Comment by frezbo** [comment, marked as answer]
just need to update talosctl version, though as per support matrix 1.31 is only available for talos 1.8 https://www.talos.dev/v1.8/introduction/support-matrix/

**Reply by erustyrannus** [reply]
Thanks! I will make sure I check the matrix next time.
